examples/ring_with_tracing/src/ring_with_tracing/main.clj

```diff
@@ -0,0 +1,53 @@
+(ns ring-with-tracing.main
+  (:require
+    [integrant.core :as ig]
+    [ring.adapter.jetty :refer :all]
```

Comment:
Can the `:refer :all` be removed to be more specific?

Reply:
Thank you. I'll fix.